common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RetryingBlockTransferor.java

```diff
@@ -192,8 +200,14 @@ private synchronized void initiateRetry() {
   private synchronized boolean shouldRetry(Throwable e) {
```

Comment:
Should the doc above be updated? Since we will also retry for SaslTimeoutException.